src/includes/class-alma-wc-settings-helper.php

```diff
@@ -31,6 +31,15 @@ public static function default_variable_price_selector() {
 		return $selector;
 	}
 
+	/**
+	 * Gets the default js event to trigger the change of product variation on product page.
+	 *
+	 * @return string
+	 */
+	public static function default_check_variations_event() {
+		return 'check_variations';
```

Comment:
you can set this as `src/includes/class-alma-wc-settings.php` constant.

this method seems not necessary

Reply:
True. But removing it would make the code less homogeneous.
And finally I realize that it is not useful to offer the merchant to choose the name of an event to be coded in javascript, but that we could impose this event name (for example "custom_check_variations") and indicate to the merchant that he must use it if he wishes.
Let's talk about is soon.

Reply:
ok, done.